newIDE/app/src/AssetStore/InstallAsset.js

```diff
@@ -246,7 +246,8 @@ export const addAssetToProject = async ({
             parameter.defaultValue,
             true,
             true,
-            true
+            true,
+            false
```

Comment:
Good opportunity I think to use "named parameters" to make the code clearer: 
```
/*matchCase=*/ true, 
/*inConditions=*/ true,
...
```

Ideally we would use an option object rather than a set of booleans (it's not a great style of code) but well!